hbase-server/src/main/java/org/apache/hadoop/hbase/master/AssignmentManager.java

```diff
@@ -1661,6 +1661,17 @@ public void regionOffline(final HRegionInfo regionInfo) {
     regionOffline(regionInfo, null);
   }
 
+  /**
+   * Marks the region as offline. In addition whether removing it from
+   * replicas and master in-memory server holding map.
+   * <p>
+   * @param regionInfo
+   * @param force
```

Comment:
let's add some descriptive text about why `force` would be used. Specifically that no known use case should have to use it except hbck, which desires to force a region offline and not have it ever be reopened on another server.

Reply:
updated. thanks!